services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/AmenitiesTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class AmenitiesTest extends TestUtils {
+
+  @Test
+  public void sanity(){
+    assertNotNull(getDefault());
+  }
+
+  private Amenities getDefault() {
+    return Amenities.builder()
+        .type("restaurant")
+        .name("starbucks")
+        .build();
+  }
+
+  @Test
+  public void testSerializableObject() throws Exception {
```

Comment:
We should also remove `test` prefix from tests names, it's implicit with the usage of `@Test` annotation.

Same thing applies to other tests below 👇